### DISCUSSION TITLE
BCC > 50

### DISCUSSION THREAD

**Discussion by fernandoSynetic** [postalserver/postal#2693] [Help with using Postal]
Hi, I have this error "The maximum number of BCC addresses has been reached (maximum 50)".

How can I increase that limit?

Thanks!

**Comment by willpower232** [comment]
You would have to change the code or submit one message per email address to Postal. Arguably, you would be adding the same amount to the queue either way.

https://github.com/postalserver/postal/blob/2f62baa238fc1102706ee4acf079b7a876b05283/app/models/outgoing_message_prototype.rb#L113-L123
